TTS/bin/train_vocoder_gan.py

```diff
@@ -495,7 +496,11 @@ def main(args):  # pylint: disable=redefined-outer-name
     optimizer_gen = getattr(torch.optim, c.optimizer)
     optimizer_gen = optimizer_gen(model_gen.parameters(), lr=c.lr_gen, **c.optimizer_params)
     optimizer_disc = getattr(torch.optim, c.optimizer)
-    optimizer_disc = optimizer_disc(model_disc.parameters(), lr=c.lr_disc, **c.optimizer_params)
+
+    if c.discriminator_model == 'hifigan_discriminator':
```

Comment:
Why do you need this differentiation?

Reply:
I had to do this so that the parameters were in the same order as the official implementation. Thus, it is possible to continue training a model trained in the official implementation. Otherwise, the optimizer breaks.

Reply:
Yes, it's necessary. Otherwise, the training breaks.